src/ln/channelmanager.rs

```diff
@@ -6636,20 +6636,28 @@ mod tests {
 			_ => panic!("Unexpected event"),
 		}
 
+		reconnect_nodes(&nodes[0], &nodes[1], (false, true), (0, 0), (0, 0), (0, 0), (0, 0), (false, false));
```

Comment:
What was the kind of bug ? (for the record if one of us beach on it again)

Reply:
I saw some issue running rust-lightning-bitcoincorerpc opening an outbound channel then restarting when the remote side had sent its funding_locked but we had not yet sent ours.